src/battle_controller_opponent.c

```diff
@@ -637,6 +617,184 @@ static void OpponentHandleChooseMove(u32 battler)
     }
 }
 
+static const u16 naturalEnemies[][3] =
+{
+    // Attacker              // Target
+    {SPECIES_ZANGOOSE,       SPECIES_SEVIPER},
+    {SPECIES_HEATMOR,        SPECIES_DURANT},
+    {SPECIES_SABLEYE,        SPECIES_CARBINK},
+    {SPECIES_MAREANIE,       SPECIES_CORSOLA},
```

Comment:
I don't think this issue is specific to this PR but with the species refactor, if any of those species are disabled the code wont compile, I believe.

Reply:
Nah, that shouldn't be an issue. The species refactor doesn't remove the species defines.